specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
+      "get": {
+        "summary": "Gets subscription core usages for location, for a given type of quota.",
+        "description": "Gets the usage in cores for a subscription in a given location, for a specific quota type.",
+        "operationId": "GetUsagesInLocation_list",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "name": "quotaType",
+            "description": "The type of quota to get usages for",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/CsmUsageQuotaCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get usages in location for subscription": {
+            "$ref": "./examples/GetUsagesInLocation.json"
+          }
+        }
+      },
+      "x-ms-pageable": {
```

Comment:
This x-ms-pageable is causing funny sounding linter failures because it is added at the wrong level. It should be part of the response definition. You've added it at the REST API verb level.